### PR TITLE
Fix unformatted Skript system messages 

### LINKED ISSUE #7103: Operators Reloading bug
### Skript/Server Version

```markdown
[12:16:07 INFO]: [Skript] Skript's aliases can be found here: https://github.com/SkriptLang/skript-aliases
[12:16:07 INFO]: [Skript] Skript's documentation can be found here: https://docs.skriptlang.org/
[12:16:07 INFO]: [Skript] Skript's tutorials can be found here: https://docs.skriptlang.org/tutorials
[12:16:07 INFO]: [Skript] Server Version: 1.21.1-77-4ff58c4 (MC: 1.21.1)
[12:16:07 INFO]: [Skript] Skript Version: 2.9.2 (selfbuilt-unknown)
[12:16:07 INFO]: [Skript] Installed Skript Addons: 
[12:16:07 INFO]: [Skript]  - DiSky v4.19.0
[12:16:07 INFO]: [Skript]  - skript-reflect v2.5.1 (https://github.com/SkriptLang/skript-reflect)
[12:16:07 INFO]: [Skript] Installed dependencies: None
```

### Bug Description

Ever since #6780 was merged, it has caused messages to be bugged and not formatted properly. Seems to be an issue with redirectingloghandler.

### Expected Behavior

The messages to be formatted properly.

### Steps to Reproduce

use a skript jar from dev/feature, reload a skript, boom broken messages
(optional) use a skript jar from dev/feature, reload a skript, make sure there is an error or 2 or however many you want, boom even more broken messages

### Errors or Screenshots

**No Errors + Ingame**
image
**Errors + Ingame**
image
**No Errors + Console**
image
**Errors + Console**
image

### Other

_No response_

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this report.

### PR DESCRIPTION
**Pull request by sovdeeth**
### Description

Fixes #7103 

---
**Target Minecraft Versions:** any 
**Requirements:** none 
**Related Issues:** #7103

### CODE REVIEW

**Review by Efnilite — APPROVED** [review]

File: src/main/java/ch/njol/skript/log/LogEntry.java [hunk 1]
```diff
@@ -171,7 +170,7 @@ public String toFormattedString() {
 			from = ChatColor.GRAY + "   " + from + "\n";
 
 		return
-			String.format(lineInfoMsg, String.valueOf(node.getLine()), c.getFileName()) + // String.valueOf is to convert the line number (int) to a String
+			String.format(lineInfoMsg, node.getLine(), c.getFileName()) + // String.valueOf is to convert the line number (int) to a String
```
Comment (cheeezburga) [inline src/main/java/ch/njol/skript/log/LogEntry.java:173]: Does the comment still need to be here?

**Review by cheeezburga — APPROVED** [review]

File: src/main/java/ch/njol/skript/log/LogEntry.java (hunk 1 above)
Comment (sovdeeth) [reply src/main/java/ch/njol/skript/log/LogEntry.java:173]: oh i didn't scroll that far haha

File: src/main/java/ch/njol/skript/log/LogEntry.java [hunk 2]
```diff
@@ -171,7 +169,7 @@ public String toFormattedString() {
 			from = ChatColor.GRAY + "   " + from + "\n";
 
 		return
-			String.format(lineInfoMsg, String.valueOf(node.getLine()), c.getFileName()) + // String.valueOf is to convert the line number (int) to a String
+			String.format(lineInfoMsg, node.getLine(), c.getFileName()) +
```
Comment (APickledWalrus) [inline src/main/java/ch/njol/skript/log/LogEntry.java:172]: Is this safe? Doesn't the lang message need to be changed to not use %s as the format specifier now?

Comment (sovdeeth) [reply src/main/java/ch/njol/skript/log/LogEntry.java:172]: I tested it to make sure and it works fine, so I think java's smart enough to convert it.

**Review by APickledWalrus — APPROVED** [review]

**Review by Efnilite — APPROVED** [review]

**Review by cheeezburga — APPROVED** [review]

**Review by UnderscoreTud — APPROVED** [review]

**Pull request merged**